.github/dependabot.yml

```diff
@@ -31,3 +31,36 @@ updates:
       - dependency-name: "org.eclipse.jetty:*" # Jetty 11 moved some code to a different package, updated the Servlet version which requires more work
         versions:
           - ">= 11.0"
+  # 2.39
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+      time: "02:17" # GitHub says 'High load times include the start of every hour'
+      timezone: "Europe/Oslo"
+    pull-request-branch-name:
+      separator: "-"
+    commit-message:
+      prefix: "chore"
+      include: "scope"
+    target-branch: "2.39"
+  - package-ecosystem: "maven"
+    directory: "/dhis-2"
+    schedule:
+      interval: "daily"
+      time: "02:17" # GitHub says 'High load times include the start of every hour'
+      timezone: "Europe/Oslo"
+    pull-request-branch-name:
+      separator: "-"
+    commit-message:
+      prefix: "chore"
+      include: "scope"
+    labels:
+      - "dependencies"
+      - "java"
+      - "run-api-tests"
```

Comment:
Do we need `run-api-test` label? I think we are using inverse logic in the workflow https://github.com/dhis2/dhis2-core/blob/3808769b1edc43b88fff523cf870e7f65524c6c2/.github/workflows/run-api-tests.yml#L23

I found that sometimes api-tests are not running and I need to remove that label and then put it back to have them running, not sure why

Reply:
You are right, we do not need it. Maybe this changed at some point from opt-in to opt-out 🤷🏻 

I also experienced what you describe. I am not sure if this issue has to do with https://github.com/dhis2/dhis2-core/blob/37afd8bc755c2f5ff5c3cec179c2e6edb4b9a610/.github/workflows/run-api-tests.yml#L3-L5 The event when the workflows are triggered.

Reply:
Will remove the label 👌🏻